mafiasi/base/management/commands/loadpasswd.py

```diff
@@ -53,7 +52,21 @@ def handle(self, *args, **options):
                 passwd_entry.save()
                 updated_entries += 1
 
+        # After importing the new users, unset email addresses of conflicting users
+        usernames = Mafiasi.objects.raw("""SELECT base_mafiasi.id FROM base_mafiasi 
```

Comment:
Why selecting `.id`? The username is stored in the column ` username     | character varying(150)   |           | not null | `